### PR TITLE
TC02 - User can't login with blank "Username" textbox

### PR DESCRIPTION
**Pull request by cvan8397**

### CODE REVIEW

File: src/test/java/tests/TC02.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package tests;
+
+import helper.Constant;
+import models.Account;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_objects.HomePage;
+import page_objects.LoginPage;
+
+public class TC02 extends TestBase {
```
Comment (tanmle) [inline src/test/java/tests/TC02.java]: name your class with a more meaningful name

File: src/test/resources/testNG.xml [hunk 2]
```diff
@@ -3,7 +3,8 @@
     <test name="Chrome Test">
         <parameter name="browser" value="chrome"/>
         <classes>
-            <class name="tests/TC01.java" />
+            <class name="tests.TC01" />
+            <class name="tests.TC02" />
         </classes>
     </test>
 </suite>
```
Comment (tanmle) [inline src/test/resources/testNG.xml]: break line.

**Review by tanmle — CHANGES_REQUESTED** [review]

**Comment by cvan8397** [comment]
@tanmle I have fixed my code, please review again and let me know if any problems

File: src/test/java/tests/TC02_UserCanNotLoginWithBlankUsername.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import page_objects.LoginPage;
+
+public class TC02_UserCanNotLoginWithBlankUsername extends TestBase {
+    HomePage homePage = new HomePage();
+    LoginPage loginPage = new LoginPage();
+    String blank = "";
+
+    @Test(description = "User can't login with blank 'Username' textbox")
+    public void testCase02() {
+        Account account = new Account(blank, Constant.PASSWORD);
```
Comment (tanmle) [inline src/test/java/tests/TC02_UserCanNotLoginWithBlankUsername.java]: you can enter `""` directly, we don't need a variable for just a simple blank

Comment (cvan8397) [reply src/test/java/tests/TC02_UserCanNotLoginWithBlankUsername.java]: @tanmle I have fixed my code, please review it again

**Review by tanmle — APPROVED** [review]

**Pull request merged**